### ISSUE TITLE
Popup window on hovering the tray icon shows terminated processes

### ISSUE THREAD

**Issue by SanderBouwhuis** [winsiderss/systeminformer#2164]
Bug:
The hover window for the tray icon is extremely confusing.
It shows processes which are already terminated with usage percentages.

Solution:
Only show processes which are actually running.
Alternatively, clearly mark terminated processes with a different colour or background.

**Comment by dmex** [comment]
> It shows processes which are already terminated with usage percentages.
> Only show processes which are actually running.

At the bottom of the popup window there's a settings icon where you can enable the `Refresh Automatically` option. If the option is disabled then the popup window can't remove terminated processes:

![image](https://github.com/user-attachments/assets/20aad08a-5109-4592-8f18-e0cfe0471c09)

**Comment by dmex** [comment]
@SanderBouwhuis 

Did you change the highlighting duration? Your video shows terminated processes remaining visible more than 1 second and those settings also determine when terminated processes are removed from view.

**Comment by SanderBouwhuis** [comment]
@dmex 
Yes, I leave the highlight long so that I can follow terminated processes in the main window. Are you saying the bug in the tray icon is caused by that?
Please either don't look at the main windows's highlight for the tray icon, or CLEARLY mark which processes are already terminated (red or gray or italic or something like that).

**Comment by dmex** [comment]
The tray window now labels terminated processes and changes the color for entries in the latest canary builds.
